Formula/czmqpp.rb

```diff
@@ -0,0 +1,81 @@
+class Czmqpp < Formula
+  desc "C++ wrapper for czmq"
+  homepage "https://github.com/zeromq/czmqpp"
+  url "https://github.com/zeromq/czmqpp.git", :tag => "v1.2.0", :revision => "a3c38eb6e3018887a8ecabc77f3029f7edb7b2d0"
+
+  head "https://github.com/zeromq/czmqpp.git"
+
+  option :universal
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "czmq"
```

Comment:
The test nicely shows that C++11 is a requirement for this library. Please add `needs :cxx11` after this line, separated from this `depends_on` by an empty line.